src/App.js

```diff
@@ -1,25 +1,28 @@
 
 import './App.css';
-import BackgroundImage from '/Users/dmitry/Desktop/Rick-Morty/rick-morty/src/content/images/background-image.jpg'
+import BackgroundImage from './content/images/background-image.jpg'
+import BigImage from '../src/content/images/big-image.png'
 import React, {useState} from 'react'
 import Characters from '../src/components/Characters/Characters.js';
 import './App.css'
 
 function App() {
   const [activeCharacters, setActiveCharacters] = useState(false)
+  const [activeLocations, setActiveLocations] = useState(false)
+  const [activeEpisodes, setActiveEpisodes] = useState(false)
 
   const HandlerCharacters = (event) => {
     event.preventDefault();
     return activeCharacters === false? setActiveCharacters(true): setActiveCharacters(false)
   }
 
   return (
-      <div>
-         <img src={BackgroundImage} className="background-image"></img>
+      <div className='main-container'>
```

Comment:
I'd recommend using double quotes for react string props

Reply:
I will replace it everywhere. And will use only it in the future